article/app/views/fragments/liveBlogNavigation.scala.html

```diff
@@ -15,7 +15,14 @@
             }
             class="liveblog-navigation__link liveblog-navigation__link--secondary liveblog-navigation__link--secondary--newer @if(pagination.newest.isEmpty) { liveblog-navigation__link--disabled }"
             data-link-name="newest page">
-                @fragments.inlineSvg("arrow-left-double", "icon", List("liveblog-navigation__link--secondary--newer__icon"))
+                @fragments.inlineSvg(
+                    "arrow-left-double",
+                    "icon",
+                    List(
+                        "liveblog-navigation__link--secondary--newer__icon",
```

Comment:
It was pre-existing code, but you're right, it's messy and I can tidy it up as part of this change 😄